lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -280,61 +729,31 @@ func (plmgr *PluginManager) loadPlugin(pluginName string, pluginVersion string)
 // RegisterActionWithEngine : Register plugin with engine
 func (plmgr *PluginManager) registerActionWithEngine(pluginName string) error {
     if err := plmgr.clientTx.RegisterAction(pluginName); err != nil {
-        //delete(plmgr.plugins, pluginName) // TODO: Goutham : verify Is this needed instead. Can't we store failed plugins ?
-        //delete(plmgr.pluginMetadata, pluginName) // TODO: Goutham : verify Is this needed instead. Can't we store failed plugins ?
+        delete(plmgr.plugins, pluginName)        // TODO: Goutham : verify Is this needed instead. Can't we store failed plugins ?
```

Comment:
A log periodic on failed action would be sufficient.

Reply:
Periodic log is there in defer function. Removed these comments

Reply:
Added